api/portainer.go

```diff
@@ -220,6 +220,17 @@ type (
 		TLSKeyPath    string `json:"TLSKey,omitempty"`
 	}
 
+	// WebhookID represents an webhook identifier.
+	WebhookID int
+
+	// Webhook represents a url webhook that can be used to update a service
+	Webhook struct {
+		ID         WebhookID  `json:"Id"`
+		TokenData  string     `json:"Token"`      //The token used to identify the service
```

Comment:
Simply rename this to `Token` instead of `TokenData`. Also you can remove the comments next to each field as each field name is pretty explanatory.